server/controllers/index.js

```diff
@@ -63,13 +63,15 @@ router.get('/api/start', auth, (req, res) => {
     res.sendStatus(200);
 });
 
-/* Check if player is logged in */
-router.get('/api/end', auth, (req, res) => {
-    currentUsers = []
-    gameLogic.endGame();
-    req.session.destroy();
-    res.sendStatus(200);
-});
+/* Check if in dev mode, and enable end game request */
+if (!process.env.NODE_ENV) {
+    router.get('/api/end', (req, res) => {
```

Comment:
HTTP specification says that GET requests are not supposed to have side-effects.
I know GET is more convenient (because browser address bar gives you a cheeky shortcut to send GET requests).

you could do it as a POST with no payload.